omnigibson/action_primitives/starter_semantic_action_primitives.py

```diff
@@ -171,10 +168,10 @@ def _assemble_robot_copy(self):
                 self._set_prim_pose(copy_mesh, mesh_copy_pose)
 
     def _set_prim_pose(self, prim, pose):
-        translation = lazy.pxr.Gf.Vec3d(*np.array(pose[0], dtype=float))
+        translation = lazy.pxr.Gf.Vec3d(*(th.tensor(pose[0], dtype=th.float32).tolist()))
         prim.GetAttribute("xformOp:translate").Set(translation)
-        orientation = np.array(pose[1], dtype=float)[[3, 0, 1, 2]]
-        prim.GetAttribute("xformOp:orient").Set(lazy.pxr.Gf.Quatd(*orientation))
+        orientation = th.tensor(pose[1], dtype=float)[[3, 0, 1, 2]]
```

Comment:
maybe we should unify `dtype=float` and `dtype=th.float32`? Currently `dtype=float` is `float64`.